### PR TITLE
Add sequence numbers to keep-alive packets

### PR DESCRIPTION
**Pull request by lynxis**
Some ISP killing duplicated packets. A sequence number
prevents this

### CODE REVIEW

**Comment by kostko** [comment]
There are ISPs that do this? Why?

Anyway, the patch looks good, just please add some comments explaining why a sequence number is needed in a keepalive message.

**Comment by lynxis** [comment]
> There are ISPs that do this? Why?
I would like to know that. cable ISP and mobile network operators doing such miserable things.

**Comment by kostko** [comment]
I would add that to both, the broker and the client in the place where the keepalive message is generated.

**Comment by lynxis** [comment]
done

File: broker/l2tp_broker.py [hunk 1]
```diff
@@ -393,6 +393,14 @@ def __init__(self, manager, port):
     self.next_session_id = 1
     self.id = -1
     self.keep_alive()
+    self.__keepalive_seqno = 0
```
Comment (mitar) [inline broker/l2tp_broker.py]: Waw. Why `__`? This is really unnecessray.

File: broker/l2tp_broker.py [hunk 2]
```diff
@@ -393,6 +393,14 @@ def __init__(self, manager, port):
...
+
+  @property
+  def _keepalive_seqno(self):
```
Comment (mitar) [inline broker/l2tp_broker.py]: Also it is great to have verbose names. This could be `nextSequenceNumber` and not reason for a property.

Comment (mitar) [inline broker/l2tp_broker.py]: (So that it is clear that there are side-effects.)

**Comment by mitar** [comment]
I updated it in `master`.

**Pull request merged**